styles/styles.css

```diff
@@ -0,0 +1,115 @@
+@import url('https://fonts.googleapis.com/css?family=Noto+Sans' rel="stylesheet");
+
+.container {
+  font-family: 'Noto Sans', sans-serif;
+  text-align: center;
+  width: 100%;
+  }
+
+ul {
+  list-style-type: none;
+}
+
+nav ul {
+  text-align: left;
+  font-weight: bold;
```

Comment:
The font-weight property definitely makes your text easier on the eyes.  (I totally forgot it existed, so nice job!)